src/fundamental/macro-fundamental.h

```diff
@@ -173,7 +173,7 @@
 #else
 #  define ABS(a) __builtin_imaxabs(a)
 #endif
-assert_cc(sizeof(intmax_t) <= sizeof(long long));
+assert_cc(sizeof(long long) == sizeof(intmax_t));
```

Comment:
hmm, we have another instance of this check in `boot/efi-string.c`, where the case of `sizeof(long long) != sizeof(intmax_t)` is even handled. Maybe drop those if you're doing this?

Reply:
I'll submit a follow-up.